payment_payfip/__manifest__.py

```diff
@@ -1,34 +1,24 @@
 {
-    'name': "Intermédiaire de paiement PayFIP",
-    'version': '11.0.22.02.25',
-    'summary': """Intermédiaire de paiement : Implémentation de PayFIP""",
-    'author': "Horanet",
-    'website': "http://www.horanet.com/",
-    'license': "AGPL-3",
-    'category': 'Accounting',
-    'external_dependencies': {
-        'python': [
-            'openupgradelib',
-        ]
-    },
-    'depends': [
-        'payment'
+    "name": "Intermédiaire de paiement PayFIP",
+    "version": "16.0.0.1.1",
+    "summary": """Intermédiaire de paiement : Implémentation de PayFIP""",
+    "author": "Horanet",
+    "website": "https://www.yotech.pro/",
+    "license": "AGPL-3",
+    "maintainer": "Yotech, Horanet",
+    "category": "Accounting",
```

Comment:
```suggestion
    "category": "Accounting/Payment Providers",
```
Les intermédiaires de paiements de Odoo ont tous cette catégorie